ci/steps/component_descriptor.py

```diff
@@ -225,15 +288,78 @@ def virtual_machine_image_resource(
     )
 
 
+def _image_rootfs_resource(
+    release_manifest: glci.model.OnlineReleaseManifest,
+    cicd_cfg,
+    effective_version: str,
+):
+    labels = [
+        cm.Label(
+          name='gardener.cloud/gardenlinux/ci/build-metadata',
+          value={
+              'modifiers': release_manifest.modifiers,
+              'buildTimestamp': release_manifest.build_timestamp,
+              'debianPackages': [p for p in _virtual_image_packages(release_manifest, cicd_cfg)],
+          }
+        ),
+    ]
+
+    rootfs_file_path = release_manifest.path_by_suffix('rootfs.tar.xz')
+    bucket_name = cicd_cfg.build.s3_bucket_name
+
+    resource_access = cm.S3Access(
+        type=cm.AccessType.S3,
+        bucketName=bucket_name,
+        objectKey=rootfs_file_path.s3_key,
+    )
+
+    resource_type = 'application/tar+vm-image-rootfs'
+
+    return cm.Resource(
+        name='rootfs_tar_xz',
```

Comment:
nit: how about naming this just `rootfs`?

Reply:
done